tests/shell/shell_recorder/kdb_ls.esr

```diff
@@ -0,0 +1,19 @@
+Storage: dump
+Mountpoint: system/tmount/kdb_ls
+DiffType: File
+
+< kdb mount $File $Mountpoint $Storage
+
+< kdb set $Mountpoint/test bla
```

Comment:
Ahh you've already seen it, i was still messing around with the shell recorder. What do you mean with tutorial syntax? I haven't found anything about that in the Readme. Do you mean i should write a tutorial how to use kdb ls and then use this tutorial wrapper?

I'm not really sure how the shellrecorder has to be used correctly yet. I've created my esr script which contains some commands that will be done on a temporary mountpoint to avoid polluting the host system. I think the extra mount was totally unnecessary as it seems the shell recorder already takes care about that, so i removed it. Also i moved the mountpoint to user where everyone can write to without having root rights.

Then i execute this script via the shellrecorder and grab the protocol it produces (kdb_lsprotocol). After checking if the output of each command was as expected in the protocol, the test can be executed by running the shellrecorder script again and comparing the output to the stored protocol file, which should match (and did match when i execute the test directly via `./shell_recorder.sh kdb_ls.esr kdb_lsprotocol`).

But does the test execution (make_run) even compare the output against the protocol file? Inspecting the CMakeFile made me doubt that. Do i have to copy the expected output back in the script file (kdb_ls.esr) and check it everywhere with the STDOUT command? That would be much more cumbersome than simply comparing against the protocol.

And in all the .esr files it uses the kdb commands directly, but if i don't use e.g. kdb ls instead of just ls, it will fail as it actually tries to use my system's ls command and not the one of kdb.